sparkly/reader.py

```diff
@@ -330,10 +330,13 @@ def _resolve_elastic(self, parsed_url, parsed_qs):
         if 'fields' in parsed_qs:
             kwargs['fields'] = parsed_qs.pop('fields').split(',')
 
+        path_segments = parsed_url.path.split('/')
+        es_type = path_segments[2] if len(path_segments) > 2 else None
+
         return self.elastic(
             host=parsed_url.netloc,
-            es_index=parsed_url.path.split('/')[1],
-            es_type=parsed_url.path.split('/')[2],
+            es_index=path_segments[1],
+            es_type=es_type,
```

Comment:
Should we use `path_segments` here as well? Just to be consistent with how we set `es_index`